polkadot/node/subsystem-bench/src/subsystem-bench.rs

```diff
@@ -76,12 +76,31 @@ struct BenchCli {
 	/// Maximum remote peer latency in milliseconds [0-5000].
 	pub peer_max_latency: Option<u64>,
 
+	#[clap(long, default_value_t = false)]
+	/// Enable CPU Profiling
```

Comment:
This needs to be more explicit, we are using pyroscope.

Reply:
Ok, but with default to fast run it like we do with Prometheus